BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditSSHKeyItem/SSHKeyItemState.swift

```diff
@@ -0,0 +1,19 @@
+import BitwardenSdk
+import Foundation
+
+// MARK: - SSHKeyItemState
+
+/// The state for an SSH key item.
+struct SSHKeyItemState: Equatable, Sendable {
```

Comment:
🎨 We are super inconsistent about it in the codebase, but I personally prefer to treat acronyms as words in and of themselves when doing casing, because it makes things a little easier to read (even though control-arrow movement now seems to handle it correctly in Xcode). So I would call this `SshKeyItemState`. Again, we're inconsistent and there are good arguments either way, so probably a moot point.